android-components/components/compose/cfr/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,7 @@
 <resources>
     <!-- Content description (not visible, for screen readers etc.): Description for the close button of a Contextual Feature Recommendation Popup. -->
     <string name="mozac_cfr_dismiss_button_content_description">Dismiss</string>
+
+    <!-- Content description: Text for a11y to be dictated when a CFR is displayed -->
+    <string name="mozac_cfr_a11y_title">Firefox notification</string>
```

Comment:
Just caught this. Since this is AC, we shouldn't have any reference to Firefox here. Let's figure out a string to have in AC as the default and then pass-in a string through fenix with a parameter for the browser name.

[example usage](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/FeltPrivacyInfoCard.kt#60)